From 6d2d806944f3045796db45b488c693e3beffc3cc Mon Sep 17 00:00:00 2001
From: Ribesg <Ribesg@yahoo.fr>
Date: Fri, 4 Apr 2014 20:06:47 +0200
Subject: [PATCH] Fix mc-dev files.

* Adds missing 'throws' declaration in PacketPlayOutMapChunk

Modified files:
* src/main/java/net/minecraft/server/PacketPlayOutMapChunk.java

diff --git a/src/main/java/net/minecraft/server/PacketPlayOutMapChunk.java b/src/main/java/net/minecraft/server/PacketPlayOutMapChunk.java
index 0239d12..0af58aa 100644
--- a/src/main/java/net/minecraft/server/PacketPlayOutMapChunk.java
+++ b/src/main/java/net/minecraft/server/PacketPlayOutMapChunk.java
@@ -44,7 +44,7 @@ public class PacketPlayOutMapChunk extends Packet {
         return 196864;
     }
 
-    public void a(PacketDataSerializer packetdataserializer) {
+    public void a(PacketDataSerializer packetdataserializer) throws IOException {
         this.a = packetdataserializer.readInt();
         this.b = packetdataserializer.readInt();
         this.inflatedBuffer = packetdataserializer.readBoolean();
--
